README.md

```diff
@@ -1,94 +1,46 @@
 # A Reference Process for OSS Projects on GitHub
 
-This is an attempt to document similarities in in a number of our OSS projects hosted on GitHub. 
+This is an attempt to document practices common to a number of OSS projects we have on GitHub. 
```

Comment:
@damithc 

> ... has the ambition to cater for a wider community, if the wider community accepts it.

I have no qualms about being ambitious - in fact I'm all for it - my concern here is that a large part of this repo's content (those that I can identify: naming conventions, coding standards, community structure) is very specific to our NUS-OSS projects, and thus its validity for the larger OSS community is questionable. If we can make the contents of this repo really generic, e.g. "Your project should have a well-defined convention for issue titles, PRs, etc." (generic) instead of "Here are the conventions you should follow" (specific), then we can justify the generic-ness of the process better. Also, even generic problems like "Why have conventions for ...?" can be subjected to debates, never mind when the conventions are specific.

Where does this lead to? The only way I see out of this is to maintain another process repo in `nus-oss` that contains our specific guidelines. It becomes something like this:
```
oss-generic: can be used by the OSS community at large
    ^
    |
    | inherits
    |
nus-oss: to be used by NUS-OSS projects, e.g. TM, SE, PL; contains specific guidelines
    ^
    |
    | inherits
    |
individual projects
```
This way, the individual projects stay independent from the name "NUS" (unless following `nus-oss` guidelines already breaches it), we have a repo (`nus-oss/process`) containing processes common to all our projects, and we have a repo for generic processes that others can use, with no pollution (for the lack of better word) from our projects-specific guidelines.